README.md

```diff
@@ -17,5 +17,88 @@ This utility takes config.yaml as input. Configuration for each app is self-cont
 
 ## Contributing
 1.Use vim or InteliJUltimate or pycharm or visualstudio code.
+### Vim
 
+Following are the vimrc setting used to create this utility. Use it if you are fan of Vim.
```

Comment:
IDE/Editor-specific setup.  It is a good candidate to set up as a template for repos here https://github.com/aicore/core.ai/issues/259 .

Reply:
In my opinion, having a vim configuration is would help someone to start with the development. Once we have the traction, we could move it to the right place, as you suggested.